packages/ui/src/components/List/list.tsx

```diff
@@ -0,0 +1,79 @@
+import React, { useMemo } from 'react'
+import { List as DefaultList, ListProps as DefaultListProps } from 'antd'
+import classNames from 'classnames'
+import { Typography, TypographyTextProps } from '@open-condo/ui/src'
+import { ListItemProps as DefaultListItemProps } from 'antd/es/list'
+
+const LIST_CLASS_PREFIX = 'condo-list'
+const ELLIPSIS_WORD_LENGTH = 80
+
+type ListDataSource = { label: string, value: string | number, valueType?: TypographyTextProps['type'] }
```

Comment:
I would expect valueType to be number, string and so on...How about more clear name for specifying a specific text type?